src/configuration/mod.rs

```diff
@@ -115,6 +115,23 @@ impl From<Configuration> for dogstatsd::Options {
     }
 }
 
+fn get_env_tags() -> Vec<String> {
+    let mut tags = vec![];
+    if let Ok(part_of) = std::env::var("KUBE_APP_PART_OF") {
+        tags.push(format!("kube_app_part_of:{}", part_of));
+    }
+    if let Ok(managed_by) = std::env::var("KUBE_APP_MANAGED_BY") {
+        tags.push(format!("kube_app_managed_by:{}", managed_by));
+    }
+    if let Ok(version) = std::env::var("KUBE_APP_VERSION") {
+        tags.push(format!("kube_app_version:{}", version));
+    }
+    if let Ok(instance) = std::env::var("KUBE_APP_INSTANCE") {
+        tags.push(format!("kube_app_instance:{}", instance));
+    }
+    tags
+}
+
```

Comment:
This seems like it could really blow up the cardinality, are ops really sure about this?